src/components/SelectTypeQuestion/SelectTypeQuestion.tsx

```diff
@@ -112,7 +112,6 @@ function SelectTypeQuestion() {
           backButtonHandler={previousQuestion}
           showBackButton={!state?.quiz.isFirstQuestion}
           ctaButtonText={question?.cta_text}
-          instructions={instructions}
```

Comment:
This isn't used anymore so just removed it.

Reply:
Also created this story to create github action to check types to catch things like this later https://linear.app/constructor/issue/CSL-2626/os-ui-library-add-github-action-to-check-types